packages/react/src/components/Card/CardMedia/Video.js

```diff
@@ -39,14 +61,15 @@ function CardVideo (props) {
     playsInline,
     ...restProps
   } = props
-  const [playing, setPlaying] = useState(autoPlay)
+  const [isPlaying, setPlaying] = useState(autoPlay)
```

Comment:
I think it's good practice for the _setter_ to match the value, eg. `setIsPlaying`

Reply:
true, I forgot that!